client/session.go

```diff
@@ -1730,6 +1738,7 @@ func (s *session) streamBlocksBatchFromPeer(
 	// Parse and act on result
 	for i := range result.Elements {
 		if i >= len(batch) {
+			m.fetchBlockError.Inc(1)
```

Comment:
Do we want to distinguish errors for batch of blocks vs. individual block errors? Right now feels like `fetchBlockError` is used for both